index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+        <header class="header">
+            <p class="logo">My Logo</p>
+            <nav class="menu"><img src="image/Union.svg" alt="Menu"></nav>
+        </header>
+        <main>
+            <section class="background">
+        <h2 class="head-title">I’m Mariane <br>
+            Glad to see you!</h2>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] This message is a call for action. Let's add the `a` tag to it so the user can navigate from the main page to another page.